web/src/components/CardForm.js

```diff
@@ -31,15 +36,19 @@ const CardForm = ({
             )}
             {title}
           </h1>
-          <form onSubmit={onSave || (() => {})}>
+          <form onSubmit={(canSubmit && onSave) || formSubmitNoop}>
```

Comment:
Need to suppress the default form behavior if submitting is disabled or there's no `onSave`.  Otherwise it just reloads the page and...  I don't know why we'd ever want that.